modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -1,59 +1,60 @@
-= Policy Level
+= Resource Level Policies
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: policy, custom, ootb, offline, resource level
 
-Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on the criteria. These are called _resource-level policies_. 
+Policies are by default applied to the entire API, unless configured otherwise. However, you can implement an additional level of granularity for policies, in which access is controlled based on a criteria. These are called _resource-level policies_. 
 
 At the resource level of granularity, policies are applied to only those requests that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
 
 With Mule 4, resource-level policy support now includes HTTP-based APIs along with RAML APIs, as opposed to support for only RAML APIs in Mule 3. A policy applied at a resource level affects all HTTP methods within the resource: PUT, GET, POST, and selected HTTP methods:
 
 image::methods-resources.png[]
 
-You can apply multiple conditions to filter your resources and HTTP methods. The URI template regex can be applied to one, many, or all the methods in your API.
+You can apply multiple conditions to filter your resources and HTTP methods using the xref:configire-uri-template-regex.adoc[URI template regex] to one, multiple, or all the methods in your API.
 
-== Usage Scenarios
+== Usage Scenarios for Implementing Resource Level Policies
 
 The uses for resource level policies are limitless. Listed below are a few examples of how you might implement the resource-level policies:
-The uses for resource-level policies are limitless. A few examples are:
 
 * Applying policies to specific resources
 * Securing a subset of an API
 * Setting different limits on resources
```

Comment:
Apply previous edits to the same text edited earlier.

Reply:
Done! Thanks!